tasks/upload.py

```diff
@@ -606,9 +606,24 @@ def _schedule_coverage_processing_task(
     ):
         checkpoints.log(UploadFlow.INITIAL_PROCESSING_COMPLETE)
 
-        do_parallel_processing = PARALLEL_UPLOAD_PROCESSING_BY_REPO.check_value(
-            identifier=commit.repository.repoid
-        ) and not delete_archive_setting(commit_yaml)
+        parallel_feature = ParallelFeature.load(upload_context.repoid)
+        if parallel_feature is ParallelFeature.EXPERIMENT and delete_archive_setting(
+            commit_yaml
+        ):
+            parallel_feature = ParallelFeature.SERIAL
```

Comment:
do you happen to know why this setting should disable the experiment? is it a problem for the fully parallel mode?

Reply:
I changed the relevant code to avoid creating a *copy* of the upload, in favor of just using the upload as it exists, provided it *does* exist and is not being deleted :-)

Parallel processing does not have that problem, as it only has a single task processing (and deleting) a raw upload.